### PR TITLE
Feature/DTGB-525: Fixed image gallery width issues + remove placeholder from hero images.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/71-pages/general/info-page/info-page--variant1.twig [hunk 1]
```diff
@@ -19,46 +19,44 @@
...
-    },
-    {
-    "src": "https://loremflickr.com/735/459/ghent?random=7",
-    "alt": "placeholder image alternative text"
-    }
-    ]
-    } %}
-  </div>
+  {% include '@image-gallery--multiple' with {
+  "items": [
```
Comment (Jeroen005) [inline styleguide/components/71-pages/general/info-page/info-page--variant1.twig]: why 😭

File: styleguide/components/71-pages/general/info-page/info-page.twig [hunk 2]
```diff
@@ -83,13 +83,11 @@
...
       } %}
 
-      <div class="image-gallery--wrapper">
-        {% include '@image-gallery' with {
-        "src": "https://loremflickr.com/735/459/ghent",
-        "alt": "placeholder image alternative text",
-        "caption": "image caption"
-        } %}
-      </div>
+      {% include '@image-gallery' with {
```
Comment (Jeroen005) [inline styleguide/components/71-pages/general/info-page/info-page.twig]: hier ook 😭

File: templates/custom/nodes/node--hero-image.html.twig [hunk 3]
```diff
@@ -72,7 +72,5 @@
 #}
 
 <figure class="hero">
-  <div class="image-wrapper" data-ratio="4:1">
-    {{ content }}
-  </div>
+  {{ content }}
```
Comment (Jeroen005) [inline templates/custom/nodes/node--hero-image.html.twig]: Kan je niet beter detecten of de content empty is?

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/71-pages/general/info-page/info-page--variant1.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/71-pages/general/info-page/info-page--variant1.twig]: Fixed.

File: styleguide/components/71-pages/general/info-page/info-page.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/71-pages/general/info-page/info-page.twig]: Fixed.

File: templates/custom/nodes/node--hero-image.html.twig (hunk 3 above)
Comment (meirege) [reply templates/custom/nodes/node--hero-image.html.twig]: Fixed.

File: templates/custom/nodes/node--hero-image.html.twig [hunk 4]
```diff
@@ -71,6 +71,8 @@
  */
 #}
 
-<figure class="hero">
-  {{ content }}
-</figure>
+{% if content %}
+  <figure class="hero">
+    {{ content }}
```
Comment (Jeroen005) [inline templates/custom/nodes/node--hero-image.html.twig]: placeholder weer toevoegen

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: templates/custom/nodes/node--hero-image.html.twig (hunk 4 above)
Comment (meirege) [reply templates/custom/nodes/node--hero-image.html.twig]: It is fixed!

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**